packages/core/strapi/lib/services/metrics/sender.js

```diff
@@ -87,6 +84,9 @@ module.exports = (strapi) => {
         groupProperties: {
           ...anonymousGroupProperties,
           projectType: strapi.EE ? 'Enterprise' : 'Community',
+          numberOfAllContentTypes: _.size(strapi.contentTypes), // TODO: V5: This event should be renamed numberOfContentTypes in V5 as the name is already taken to describe the number of content types using i18n.
+          numberOfComponents: _.size(strapi.components),
+          numberOfDynamicZones: getNumberOfDynamicZones(),
```

Comment:
not following the code, so they are not added to userProperties anymore?

Reply:
This is a bug that was present, the strapi global object didn't have the required information on the previous scope. This is not necessarily really related to the Amplitude issue, but I hope it is 😅

Reply:
aah I see, so it didnt have info of the strapi content types and compos. But why not adding them in userProperties and not only in group properties?

If I understood correctly Marco needed to filter by this properties on Users.